ppoprf/src/ppoprf.rs

```diff
@@ -209,7 +217,7 @@ impl Server {
   pub fn new(mds: Vec<u8>) -> Result<Self, PPRFError> {
     let mut csprng = OsRng;
     let oprf_key = RistrettoScalar::random(&mut csprng);
-    let mut md_pks = HashMap::new();
+    let mut md_pks = BTreeMap::new();
```

Comment:
Please document the motivation for this change in the commit message.

Is it necessary to get repeatable serialisation with the bincode crate? Are you worried sorting the keys as serialisation time is too expensive, and want insertion sort? I'd be unclear, reading the code why the extra structure was chosen here.

Reply:
Sure thing, appended "(keeps map keys in order)" to the commit message.

In order for the snapshot test to pass, we need the keys to be in a reproducible order. It seems that bincode serializes map key/values according to their order in the map. When I was testing with HashMap, the order changed upon each invocation.